GoSSB/Sources/blobs.go

```diff
@@ -131,5 +131,11 @@ func ssbBlobsAdd(fd int32) *C.char {
 	}
 	f.Close()
 
+	err = sbot.WantManager.WantWithDist(br, -1)
+	if err != nil {
+		err = errors.Wrap(err, "push: pushing blob to other peers failed")
+		return nil
```

Comment:
Should we return nil here? The blob was correctly added at this point

Reply:
We could, but I thought that it might be better to fail the whole add blob operation in this case, because it means the blob likely won't leave the user's device for a long time which doesn't meet their expectations. So better to have them retry the operation. 

I didn't test this in an offline scenario though. Let me try that now. Looks like it doesn't fail, so I think I'm ok with leaving this as-is if you are.